addons/website_blog/views/snippets/s_latest_posts.xml

```diff
@@ -22,6 +22,12 @@
     </section>
 </template>
 
+<template id="s_latest_posts_big_orizontal" name="Latest Posts Horizontal">
```

Comment:
Maybe add it in the t-snippet list in this second commit, so that third commit removes the two extra which become options ?